src/stories/control/DarkModeButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { IconButton, IconButtonProps, useColorMode, useColorModeValue } from '@chakra-ui/react'
+import { SunIcon, MoonIcon } from '@chakra-ui/icons'
+
+export interface DarkModeButtonProps extends Omit<IconButtonProps, "aria-label"> { }
+
+const DarkModeButton = ({ ...props }: DarkModeButtonProps) => {
+    const { toggleColorMode } = useColorMode()
+
+    const label = useColorModeValue('Light Mode', 'Dark Mode')
+    const iconColor = useColorModeValue('#2b2b2b', 'white')
+    const icon = useColorModeValue(<SunIcon color={iconColor} />, <MoonIcon color={iconColor} />)
```

Comment:
If `iconColor` is not used anywhere else, you could set it directly inside the icon you declare below it